spec/features/thredded/user_creates_messageboard_group_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+feature 'Creating a messageboard group' do
+  scenario 'admin creates a unique messageboard group' do
+    user = an_admin
+    user.log_in
+
+    group = set_up_a_messageboard_group
+    group.visit_new_messageboard_group_form
+    group.submit_form
+
+    expect(group).to be_created
+  end
+
+  scenario 'admin creates non unique messageboard group' do
+    user = an_admin
+    user.log_in
+
+    group = set_up_a_messageboard_group
+    group.create_messageboard_group
+    group.visit_new_messageboard_group_form
+    group.submit_form
+
+    expect(group).to be_duplicate
+  end
```

Comment:
Ok I gotcha. If that's the case the exercise should be more descriptive of what the page object is doing wrong in order to test the assertion that there's an error.

1. Change `group.submit_form` to something like `group.submit_form_with_duplicate_group_ information`
2. Change the assertion to communicate more as far as what you expect. Something like `expect(group).to have_duplicate_messageboard_group_error`